### ISSUE TITLE
mAP of 0.47 on DOTA validation set

### ISSUE THREAD

**Issue by swz30** [DetectionTeamUCAS/R2CNN_Faster-RCNN_Tensorflow#17]
Using the provided pre-trained R2CNN and config files, I am getting 0.47 mAP on validation set of DOTA. is it correct?

**Comment by yangxue0827** [comment]
The mAP of the provided model weights is around 57.3.

**Comment by yangxue0827** [comment]
make sure which model weights you used, pre-trained (just for training) or trained model (after training). 
please download [trained model](https://github.com/DetectionTeamUCAS/Models/tree/master/R2CNN_Faster-RCNN_Tensorflow) by this project, put it to output/trained_weights. @swz30

**Comment by swz30** [comment]
Hi @yangxue0827 
Thank you for the response. 

I did use your trained model and put it in the same folder which you mentioned. But I am only able to get 0.47 mAP on the DOTA validation set. 

Here I am listing the steps that I performed:

1) By using val_crop.py, I cropped the DOTA validation set into 800x800 overlapping images and also obtained the corresponding xml annotation files. 

2) Then I ran the following command:
python eval.py --img_dir='PATH TO DOTA CROPPED VALIDATION IMAGES' 
               --image_ext='.png' 
               --test_annotation_path='PATH TO XML ANNOTATION FILES OF VALIDATION IMAGES'
               --gpu='0'

The eval.py scripts provides me the mAP for both horizontal and rotated bounding boxes almost the same (i.e. 0.47) on the validation set. Am I missing something?

Also, the 3rd step of data preparation, you recommended to put the data in Pascal VOC format. But then why don't we use that tree in the evaluation, is that step redundant?  

thank you.

**Comment by yangxue0827** [comment]
step val_crop.py is not required. @swz30

**Comment by yangxue0827** [comment]
python demo_rh.py --src_folder='/PATH/TO/DOTA/IMAGES_ORIGINAL/' --image_ext='.png' --des_folder='/PATH/TO/SAVE/RESULTS/' --save_res=False --gpu='0' 

then commit the results files in tools/txt_out/@swz30

@swz30

**Comment by ssynkqtd** [comment]
> The mAP of the provided model weights is around 57.3.

Which model can achieve the mAP of 68.01 in Task1 - Oriented Leaderboard and 72.80 in Task2 - Horizontal Leaderboard.

**Comment by yangxue0827** [comment]
Sorry, my improvement method is preparing for submission, so the code will not be open source for the time being. @ssynkqtd

**Comment by heping0228** [comment]
@swz30 I use the trained=weights provided by the author and I get the mAP is 39%,I also crop the val set,do you know the reason.
